### PR TITLE
feat: Improve error and debug messaging

### PR DESCRIPTION
**Pull request by jirihnidek**
* Improved messaging in various cases (mostly related to D-Bus methods and signals)
* All changes comes from closed PR #157 (running yggdrasil as non-root user)

### CODE REVIEW

**Comment by jirihnidek** [comment]
packit did all checks in 25 minutes ... :-1:

File: internal/work/dispatcher.go [hunk 1]
```diff
@@ -89,23 +89,28 @@ func (d *Dispatcher) Connect() error {
...
 	}
 
 	log.Infof("exported /com/redhat/Yggdrasil1/Dispatcher1 on bus")
 
 	// Add a match signal on the
 	// org.freedesktop.DBus.Properties.PropertiesChanged signal.
 	if err := d.conn.AddMatchSignal(dbus.WithMatchPathNamespace("/com/redhat/Yggdrasil1/Worker1"), dbus.WithMatchInterface("org.freedesktop.DBus.Properties"), dbus.WithMatchMember("PropertiesChanged")); err != nil {
-		return fmt.Errorf("cannot add signal match: %v", err)
+		return fmt.Errorf(
+			"cannot add signal match on the org.freedesktop.DBus.Properties.PropertiesChanged: %v",
```
Comment (subpop) [inline internal/work/dispatcher.go:104]: Minor grammatical request.
```suggestion
			"cannot add signal match on org.freedesktop.DBus.Properties.PropertiesChanged: %v",
```

File: internal/work/dispatcher.go [hunk 2]
```diff
@@ -89,23 +89,28 @@ func (d *Dispatcher) Connect() error {
...
+			err,
+		)
 	}
 
 	if err := d.conn.AddMatchSignal(dbus.WithMatchPathNamespace("/com/redhat/Yggdrasil1/Worker1"), dbus.WithMatchInterface("com.redhat.Yggdrasil1.Worker1"), dbus.WithMatchMember("Event")); err != nil {
-		return fmt.Errorf("cannot add signal match: %v", err)
+		return fmt.Errorf(
+			"cannot add signal match on the com.redhat.Yggdrasil1.Worker1.Event: %v",
```
Comment (subpop) [inline internal/work/dispatcher.go:111]: Minor grammatical request.
```suggestion
			"cannot add signal match on com.redhat.Yggdrasil1.Worker1.Event: %v",
```

**Review by subpop — CHANGES_REQUESTED** [review]
Looks great! I just have a couple tiny grammar changes, but otherwise this is good. Very nice improvements.

**Review by subpop — APPROVED** [review]

**Pull request merged**